content/blog/2016/2016-05-12-sf-jam-jenkins-and-azure.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "SF JAM Report: Scaling Jenkins for Continuous Delivery with Azure"
+tags:
+- jam
+- azure
+- meetup
+author: rtyler
+---
+
+A few weeks ago, my colleague link:https://github.com/brianvdawson[Brian Dawson]
+and I were invited to present on
+link:http://www.meetup.com/jenkinsmeetup/events/228524373/[Scaling Jenkins for
+Continuous Delivery with Microsoft Azure] in Microsoft's
+link:http://microsoftreactor.com/about/[Reactor space]. Azure is Microsoft's
+public cloud offering and one of the many tools available to Jenkins users for
+adding elastic compute capacity, among other things, to their build/test/deploy
+infrastructure. While Brian's and my presentations are applicable to practically
+any cloud-based Jenkins environment, Thiago Almeida and Oguz Pastirmaci from
+Microsoft were also on-hand and presented some interesting Azure-specific
+offerings like
+link:https://azure.microsoft.com/en-us/services/container-service/[Azure
+Container Service] with Jenkins.
+
+To kick-off the sessions we asked attendees a few polling questions and
+received very telling responses:
+
+* How big is your Development/IT organization?
+* What is your role?
+* By show of hands do you practice?
```

Comment:
"By show of hands do you practice" + insert: ": Continuous Integration, Continuous Delivery, and/or DevOps" + "?"

...but really should consider adding back the actual questions and responses or just pulling the whole portion on polls and observations.   Really only has value if the response %'s are included.  

I am cool with cutting it out for brevity.